colosseum/validators.py

```diff
@@ -0,0 +1,101 @@
+from . import parser
+from . import units
+
+
+def _numeric_validator(value, numeric_type, min_value, max_value):
+    error_msg = ''
+    try:
+        value = numeric_type(value)
+    except (ValueError, TypeError):
+        error_msg = "Cannot coerce {value} to {numeric_type}".format(
+            value=value, numeric_type=numeric_type.__name__)
+
+    if not error_msg and min_value is not None and value < min_value:
+        error_msg = 'Value {value} bellow minimum value {min_value}'.format(
+            value, min_value)
+
+    if not error_msg and max_value is not None and value > max_value:
+        error_msg = 'Value {value} above maximum value {max_value}'.format(
+            value, max_value)
+
+    return error_msg, value
```

Comment:
This could be just the value and a boolean, but I find that having an error message might be useful in the future?